numba/np/npyfuncs.py

```diff
@@ -1590,23 +1638,39 @@ def np_real_nextafter_impl(context, builder, sig, args):
     _check_arity_and_homogeneity(sig, args, 2)
 
     dispatch_table = {
-        types.float32: 'npy_nextafterf',
-        types.float64: 'npy_nextafter',
+        types.float32: 'numba_nextafterf',
+        types.float64: 'numba_nextafter',
     }
 
     return _dispatch_func_by_name_type(context, builder, sig, args,
                                        dispatch_table, 'nextafter')
 
 def np_real_spacing_impl(context, builder, sig, args):
+    # This is different to how NumPy does it, NumPy has a specialisation of
```

Comment:
Need to check that unit tests handle `inf` correctly.

Reply:
`inf` input appears to generally be untested in the ufunc tests, manually testing the `spacing` impl demonstrates that `spacing(np.inf)` returns `nan` regardless of the sign of the `np.inf`, this matches NumPy.

Reply:
probably worth adding this test

Reply:
Added in a28cf62.